api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -116,6 +118,83 @@
     ENABLE, FOLLOW_HADOOP_AUTH, FOLLOW_HADOOP_HTTP_AUTH, DISABLE
   }
 
+  @InterfaceStability.Evolving
+  /**
+   * Wrapper class to store SSL configuration parameters specified using the SSL-CONFIG attribute.
+   */
+  public static class SSLConfig implements Serializable
```

Comment:
LGTM with two minor comments:
- Please move SSLConfig to the end of Context.java where the attribute is declared
- import org.apache.hadoop.classification.InterfaceStability.Evolving